ee_tests/src/quickstart_analytic.spec.ts

```diff
@@ -98,38 +99,49 @@ describe('Creating new quickstart in OSIO', () => {
 
     support.writeScreenshot('target/screenshots/pipeline_icons_' + spaceName + '.png');
 
-    /* Write the Jenkins build log to stdout */
-    // await support.dumpLog(spacePipelinePage);
-
-    // TODO - Error conditions to trap
-    // 1) "View Build" link not displayed - this is issue https://github.com/openshiftio/openshift.io/issues/1194
-    // 2) Build reports error - grep for "ERROR" in Jenkins pod log
-    // 3) Timeout on build - write build duration to stdout and grep for "ERROR" in Jenkins pod log
-
-    /* Verify that the analytics report was created - TODO - expand to test of report contents */
-
-    /* TODO remove sleep statements */
     await dashboardPage.header.recentItemsDropdown.clickWhenReady();
     await dashboardPage.header.recentItemsDropdown.accountHomeItem.clickWhenReady();
     await dashboardPage.header.recentItemsDropdown.clickWhenReady();
     await dashboardPage.recentSpaceByName(spaceName).click();
-    await spaceDashboardPage.stackReportsButton.clickWhenReady(support.LONGER_WAIT);
-
-    await browser.sleep(support.DEFAULT_WAIT);
-    try {
-      await expect(spaceDashboardPage.stackReportDependencyCardTotalCount.getText())
-        .toContain(quickstart.dependencyCount.total);
-      await expect(spaceDashboardPage.stackReportDependencyCardAnalyzedCount.getText())
-        .toContain(quickstart.dependencyCount.analyzed);
-      await expect(spaceDashboardPage.stackReportDependencyCardUnknownCount.getText())
-        .toContain(quickstart.dependencyCount.unknown);
-      support.writeScreenshot('target/screenshots/analytic_report_success_' + spaceName + '.png');
-    } catch (e) {
-      support.writeScreenshot('target/screenshots/analytic_report_fail_' + spaceName + '.png');
-      throw e;
-    }
-
-    await spaceDashboardPage.analyticsCloseButton.clickWhenReady();
+    await dashboardPage.ready();
+
+    let codeBasesCard = await spaceDashboardPage.getCodeBaseCard();
```

Comment:
it's called *Codebase* in OSIO so the name of method should be `getCodebaseCard`, the same logic for variable name

Reply:
OK